[2021-09-28 15:41:48,690] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_2 2021-09-26T00:00:00+00:00 [queued]>
[2021-09-28 15:41:48,698] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_2 2021-09-26T00:00:00+00:00 [queued]>
[2021-09-28 15:41:48,698] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-09-28 15:41:48,698] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-09-28 15:41:48,698] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-09-28 15:41:48,772] {taskinstance.py:1115} INFO - Executing <Task(BashOperator): runme_2> on 2021-09-26T00:00:00+00:00
[2021-09-28 15:41:48,774] {standard_task_runner.py:52} INFO - Started process 41063 to run task
[2021-09-28 15:41:48,781] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'runme_2', '2021-09-26T00:00:00+00:00', '--job-id', '4', '--pool', 'default_pool', '--raw', '--subdir', '/home/aman/PycharmProjects/air_my/venv/lib/python3.8/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpzssb_27h', '--error-file', '/tmp/tmpbzphheru']
[2021-09-28 15:41:48,782] {standard_task_runner.py:77} INFO - Job 4: Subtask runme_2
[2021-09-28 15:41:48,913] {logging_mixin.py:109} INFO - Running <TaskInstance: example_bash_operator.runme_2 2021-09-26T00:00:00+00:00 [running]> on host shah
[2021-09-28 15:41:49,122] {taskinstance.py:1252} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=runme_2
AIRFLOW_CTX_EXECUTION_DATE=2021-09-26T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-26T00:00:00+00:00
[2021-09-28 15:41:49,122] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-09-28 15:41:49,123] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'echo "example_bash_operator__runme_2__20210926" && sleep 1']
[2021-09-28 15:41:49,130] {subprocess.py:74} INFO - Output:
[2021-09-28 15:41:49,132] {subprocess.py:78} INFO - example_bash_operator__runme_2__20210926
[2021-09-28 15:41:50,133] {subprocess.py:82} INFO - Command exited with return code 0
[2021-09-28 15:41:50,237] {taskinstance.py:1212} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=runme_2, execution_date=20210926T000000, start_date=20210928T095648, end_date=20210928T095650
[2021-09-28 15:41:50,358] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-09-28 15:41:50,469] {local_task_job.py:261} INFO - 1 downstream tasks scheduled from follow-on schedule check
